### DISCUSSION TITLE
Days Gone's texture size too small?

### DISCUSSION THREAD

**Discussion by sc1991327** [gildor2/UEViewer#260] [Q&A]
most of texture of game Days Gone, only can have very small size. command line told me get the file error:

Loading Texture2D cinnabar_01_c from package /Game/Environments/Decals/Rocks/cinnabar_01_c.uasset
FByteBulkData::Serialize: position is outside of the file (524288 bytes)
FByteBulkData::Serialize: position is outside of the file (131072 bytes)
FByteBulkData::Serialize: position is outside of the file (32768 bytes)
FByteBulkData::Serialize: position is outside of the file (8192 bytes)

![image](https://user-images.githubusercontent.com/5671048/149703718-66890d65-ac5b-41b0-809a-719ee57b888f.png)

![image](https://user-images.githubusercontent.com/5671048/149703741-b2b63f7b-719e-4245-af19-8a0a47fb892c.png)

how to solve it?

**Comment by Blenux** [comment]
What settings did you use when launching uModel?

Also this would be better on the forums where there is a game thread, you will find the following information...

> Regarding the PC version of the game: so that there are no texture reading errors (for example, a 64x64 texture on a model). You must run umodel from the folder with the main game archive i.e. [b]BendGame-WindowsNoEditor.pak[/b] then the textures are read correctly and will already be 1024x1024 or more. Good luck.

[Days Gone](https://www.gildor.org/smf/index.php/topic,6602.45.html)

**Reply by sc1991327** [reply]
![WXWorkCapture_16424661434782](https://user-images.githubusercontent.com/5671048/149851355-de84b30f-ace0-43f3-b03f-7eeee18dfc4b.png)

I test multiple settings
game detection: unreal engine 4 + days gone/UE4.11/UE4.2x
platform is PC, this game is form stream

**Comment by gildor2** [comment]
Check the log the you're opening the texture asset, most likely you'll see that some file is missing.
